deps/rabbit/src/rabbit_core_ff.erl

```diff
@@ -100,22 +104,5 @@ direct_exchange_routing_v2_migration(FeatureName, _FeatureProps, enable) ->
                                              [FeatureName, Reason]),
               {error, Reason}
     end;
-direct_exchange_routing_v2_migration(_FeatureName, _FeatureProps, is_enabled) ->
-    TableName = rabbit_index_route,
-    Enabled = rabbit_table:exists(TableName),
-    case Enabled of
-        true ->
-            %% Always ensure every node has a table copy.
-            %% If we were not to add a table copy here, `make start-cluster NODES=2`
-            %% would result in only `rabbit-1` having a copy.
-            case rabbit_table:ensure_table_copy(TableName, node(), ram_copies) of
-                ok ->
-                    ok;
-                {error, Reason} ->
-                    rabbit_log_feature_flags:warning("Failed to add copy of table ~s to node ~p: ~p",
-                                                     [TableName, node(), Reason])
-            end;
-        _ ->
-            ok
-    end,
-    Enabled.
+direct_exchange_routing_v2_migration(#ffcommand{command = post_enable}) ->
```

Comment:
You should rather use a "catch all" clause and ignore the argument. This allows us to add commands without having to update all migration functions.